Sources/MaliciousSiteProtection/Services/DataManager.swift

```diff
@@ -95,11 +101,10 @@ public actor DataManager: DataManaging {
         } catch {
             Logger.dataManager.error("Error encoding \(fileName): \(error.localizedDescription)")
             assertionFailure("Failed to store data to \(fileName): \(error)")
-            return
+            return false
         }
 
-        let success = fileStore.write(data: data, to: fileName)
-        assert(success)
+        return fileStore.write(data: data, to: fileName)
```

Comment:
It‘s probably better to make both `store` and `write` functions throwing to propagate the error up to the caller, since we rely on the result.